worker/worker_test.go

```diff
@@ -89,6 +89,11 @@ func TestWorker_GetTask(t *testing.T) {
 
 	utils.Client = &MockedClient{}
 
+	utils.GetSignature = func(payload interface{}, workerId string) []byte {
+		fakeSignature, _ := json.Marshal("FAKE-SIGNATURE")
+		return fakeSignature
+	}
+
```

Comment:
I think this code is not changing anything in the test. Could you explain it to me?

Reply:
is it clear enough now?